src/ducks/settings/GroupSettings/AccountsList.jsx

```diff
@@ -32,7 +31,11 @@ const AccountsList = props => {
       } else {
         accounts.removeById(accountId)
       }
-      await updateOrCreateGroup(client, group, router)
+      const res = await client.save(group)
+      if (res && res.data && !group.id) {
```

Comment:
nit: Better like that ?
```
const doc = res?.data
if (doc && !group.id) { 
 ....
}
```
instead of 
```
if (res && res.data && !group.id) {
const doc = res.data
...
}

Reply:
At least use optional chaining if (res?.data && !group.id) {